pandas/core/reshape/reshape.py

```diff
@@ -48,23 +48,23 @@ class _Unstacker(object):
     >>> import pandas as pd
     >>> index = pd.MultiIndex.from_tuples([('one', 'a'), ('one', 'b'),
     ...                                    ('two', 'a'), ('two', 'b')])
-    >>> s = pd.Series(np.arange(1.0, 5.0), index=index)
+    >>> s = pd.Series(np.arange(1, 5), index=index)
     >>> s
-    one  a   1
-         b   2
-    two  a   3
-         b   4
-    dtype: float64
+    one  a    1
+         b    2
+    two  a    3
+         b    4
+    dtype: int32
```

Comment:
this one will depend on the platform you are using, so to be more robust, we should probably best add the dtype to `np.arange`. I would use 'int64', as this is the default in pandas (in numpy the default is platform dependent). 
Or as an alternative use `range(1, 5)`, then pandas Series will always convert this to int64